src/core/template-dx-integration.ts

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+
+import { generateID } from "./helpers";
+import { ITemplateUpdater } from "./template-updater";
+import { ITemplateWrapperProps, TemplateWrapper } from "./template-wrapper";
+
+interface ITemplateDxIntegration {
+    render: (data: IDxTemplateData) => any;
+}
+
+interface IDxTemplateData {
+    container: any;
+    model?: any;
+    index?: any;
+    onRendered?: () => void;
+}
+
+function createDxIntegration(
```

Comment:
In fact, it creates a DevExtreme Template object, not an integration.

Reply:
Will rename it to `DxTemplate`